README.md

```diff
@@ -84,4 +84,6 @@ sequenceDiagram
     wsl-init (pid 1)-->>wsl-init (pid 1): Spawn /sbin/init
 
     vm-switch->>host-switch.exe: forward ethernet
-```
\ No newline at end of file
+```
+## wsl-proxy
+Its primary function comes into play when WSL integration is activated alongside the network tunnel. The purpose is to actively listen on `127.0.0.1:6443` and reroute all incoming requests to the upstream Kubernetes API server residing within the network namespace. The proxy achieves this by directing the incoming traffic through a Virtual Ethernet pair. Specifically, `veth-rd0` serves as the originating end listening in the default network, while `veth-rd1` acts as the terminating end within the network namespace.
```

Comment:
> and reroute all incoming requests to the upstream Kubernetes API server residing within the network namespace

It also routes non-Kubernetes things, right? (Think random containers)

Reply:
It routes anything that is destined for K8s API server at `127.0.0.1:6443`.